app/views/HistoricDateRequestPageView.scala.html

```diff
@@ -81,7 +81,7 @@ <h2 class="govuk-caption-xl" id="eori-heading">
                 @inputDate(
                     form,
                     messages("cf.historic.document.request.to"),
-                    hintText = Some(messages("cf.historic.document.request.date.hint")),
+                    hintText = Some(messages("cf.historic.document.request.date.pvat.hint")),
```

Comment:
Could you please write a test case for this change in the test class (HistoricDateRequestPageViewSpec) for FileRole C79Certificate ?

Reply:
It will be covered by me as part of 4878. Hence resolving